server/__test__/route_test/index.test.js

```diff
@@ -0,0 +1,57 @@
+const request = require("supertest");
+const app = require("../../app");
+const connection = require("../../database/config/connection");
+const build = require("../../database/config/build");
+
+beforeEach(() => build());
```

Comment:
Please, add beforeEach and AfterEach functions inside the describe suits, so this will have effect only this your tests inside this block.

Reply:
Ok :+1: